web/src/components/SocialMediaTooltip.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { ReactElement, ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled, { css } from 'styled-components'
+
+import { getSlugFromPath, POIS_ROUTE } from 'api-client'
+import { UiDirectionType } from 'translations'
+
+import { CloseIcon, FacebookIcon, MailSocialIcon, WhatsappIcon } from '../assets'
+import dimensions from '../constants/dimensions'
+import useWindowDimensions from '../hooks/useWindowDimensions'
+import { RouteType } from '../routes'
+import Tooltip from './Tooltip'
+
+type SocialMediaTooltipProps = {
+  active: boolean
+  children: ReactNode
+  shareLink: string
+  title: string
+  onClose: () => void
+  flow: 'top' | 'right'
+  direction: UiDirectionType
+  route: RouteType
+}
+
+const SHARE_BUTTON_WIDTH = 50
+const TooltipContainer = styled.div<{
+  flow: 'top' | 'right'
+  active: boolean
+  direction: UiDirectionType
+  additionalPadding: number
+}>`
+  background-color: ${props => props.theme.colors.backgroundColor};
+  padding: 8px;
+  border: 2px solid ${props => props.theme.colors.textDecorationColor};
+  width: max-content;
+  position: absolute;
+  display: flex;
+  z-index: 2000;
+  opacity: 0;
+
+  ${props =>
+    props.flow === 'top' &&
+    css`
+      bottom: ${dimensions.toolbarHeight + props.additionalPadding}px;
+      flex-flow: column-reverse;
+      margin-left: -10px;
+    `};
+
+  ${props =>
+    props.flow === 'right' &&
+    props.direction === 'ltr' &&
+    css`
+      left: ${dimensions.toolbarWidth - SHARE_BUTTON_WIDTH}px;
+    `};
+
+  ${props =>
+    props.flow === 'right' &&
+    props.direction === 'rtl' &&
```

Comment:
How about something like
```suggestion
  ${props =>
    props.flow === 'right' && (props.direction === 'ltr' ? css`/* css for LTR */` : css`/* CSS for RTL */`)};
```